pkg/features/kube_features.go

```diff
@@ -651,6 +651,15 @@ const (
 	// Add support for the HPA to scale based on metrics from individual containers
 	// in target pods
 	HPAContainerMetrics featuregate.Feature = "HPAContainerMetrics"
+
+	// owner: @zshihang
+	// alpha: v1.13
+	// beta: v1.20
+	//
+	// Allows kube-controller-manager to publish kube-root-ca.crt configmap to
+	// every namespace. This feature is separated from BoundServiceAccountTokenVolume
+	// to solve multi-server cluster upgrade issue.
```

Comment:
potentially we can omit the second sentence here that explains the upgrade issue and instead include a mention that BoundServiceAccountTokenVolume requires RootCAConfigMap.

deferring to sig-auth, though.

Reply:
agree the separation is not important, just that it is a prereq of BoundServiceAccountTokenVolume

Reply:
done